BCStudentSoftwareDevTeam/celts
# Add a new column to the CCE Minor Progress table to indicate that they requested engagement

Issue: add a new column to the CCE Minor Progress in the Minor Management table to indicate that they requested other community engagement. This would probably look pretty similar to the existing Summer Experience column.

test: 

- Switch to branch 1150_engagement
- Reset the data to ensure the role in the database (test data)
- In the bottom left corner switch the current user to  Admin: ramsayb2
- Click on Admin dropdown and choose Minor Management 
- In the CCE Minor Progress table there should be a column titled "Requested Other Engagement" By default it should have No for all the students.
- Now, you have to request Other community engagement for one or more students, here is how: Click on one of the students' names, then click on the blue button on the right corner, titled "Request other Experience", fill out the form, and submit the form 
- Go back to the Minor Mangement from the admin dropdown and for that student, it should show a "Yes" under the Request Other Engagement column. 
- Tests it multiple times and make sure it works as expected 
- review the code 


resolves issue #1150
